api/src/main/java/org/apache/iceberg/IncrementalAppendScan.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+package org.apache.iceberg;
+
+/**
+ * API for configuring an incremental table scan for appends only snapshots
+ */
+public interface IncrementalAppendScan extends Scan<IncrementalAppendScan> {
+
+  /**
+   * Refine the incremental scan with the start snapshot inclusive.
+   * <p>
+   * If the start snapshot (inclusive or exclusive) is not provided,
+   * the oldest snapshot will be used as the start snapshot.
```

Comment:
I don't think this statement is clear enough. The table's first snapshot is used. You could argue that's the "oldest" but I think it is better to be clear that you're asking to process snapshots back to the start of the table. Clarifying on the next line helps, but it exposes an internal detail about how we track history: a snapshot with no parent is the starting snapshot.

Reply:
Can you see if the latest change is more clear?